README.md

```diff
@@ -5,6 +5,7 @@
 Node 4.x (and above) is **required**. We suggest using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) to keep multiple Node versions on your system.
 
 
+
```

Comment:
Yes, I still need to update the README to reflect the new dev and build process.